### ISSUE TITLE
[Feature Request] Validate Address Support

### ISSUE THREAD

**Issue by AndroDevcd** [KeystoneHQ/Keystone-cold-app-btc#76]
Hello Keystone team, I am writing to request a feature to be able to scan a QR code of a bitcoin address and validate that that address is a part of the current wallet stored on the Keystone device. This will prevent malicious actors from displaying an invalid receive address that sends to a different wallet on say blue wallet, electrum, etc.  Would love to see this as a feature added in the future!

Thanks

**Comment by aaronisme** [comment]
Thanks for the feedback

**Comment by igorkost-ibm** [comment]
Hello Keystone team, I would also very much like to have this feature.
@aaronisme, Has it got any time schedule?

**Comment by BeeJoe** [comment]
Yes this is a vital feature of a hardware wallet. Hopefully this is not ignored.
